definitions/tools/xenosplit.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Xenosplit"
+
+baseCommand: ["/bin/bash", "Xenosplit.sh"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 20000
+      tmpdirMin: 100000
+    - class: DockerRequirement
+      dockerPull: "mgibio/xenosplit:0.5"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'Xenosplit.sh'
+        entry: |
+            set -o pipefail
+            set -o errexit
+            
+            # Filtering the bam files and preparing them for xenosplit
+            /opt/samtools/bin/samtools sort -o graftsorted.bam $1
+            /opt/samtools/bin/samtools index graftsorted.bam
+            /opt/samtools/bin/samtools view -h -F 256 -F 2048 graftsorted.bam > graftbam.bam
+            /opt/samtools/bin/samtools sort -n -o graftbam_accepted.bam graftbam.bam
+
```

Comment:
Could any of these steps be chained together to save on disk space and I/O?  maybe with something like? 
`samtools view -h -F 256 -F 2048 graftsorted.bam | samtools sort -n -o graftbam_accepted -`  I suspect that might be faster, possibly at the expense of some extra ram